apps/ia/ia-case-api/demo.yaml

```diff
@@ -2,9 +2,12 @@ apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: ia-case-api
+  annotations:
+    hmcts.github.com/prod-automated: disabled
 spec:
   values:
     java:
+      image: hmctspublic.azurecr.io/ia/case-api:pr-1425-cecad0d-20230627122305 #{"$imagepolicy": "flux-system:demo-ia-case-api"}
```

Comment:
Duplicated line, according to GitHub. The one below will need to be removed, presumably.

Reply:
Removed